packages/data-model/src/models/dictionary_categories.ts

```diff
@@ -7,7 +7,8 @@ import { submittedData } from './submitted_data.js';
 
 export const dictionaryCategories = pgTable('dictionary_categories', {
 	id: serial('id').primaryKey(),
-	activeDictionaryId: integer('active_dictionary_id'),
+	activeDictionaryId: integer('active_dictionary_id').notNull(),
+	defaultCentricEntity: varchar('defaultCentricEntity'),
```

Comment:
should this follow a consistent naming syntax?
e.g. `default_centric_entity`

Reply:
This comment will be handled in PR[#105](https://github.com/overture-stack/lyric/pull/105)
Fixed to snake case for consistency